File: test-client.rb
#!/usr/bin/env ruby
require 'dbus'
bus = DBus::SessionBus.instance
xanguli_Service = bus.service('org.xanguli.Service')
obj = xanguli_Service.object('/Xanguli')
obj.introspect
introspectable = obj['org.freedesktop.DBus.Introspectable']
puts "Introspect:\n" + introspectable.Introspect[0]
xanguli = obj['org.xanguli.Xanguli']
puts "CreateController:\n" + xanguli.CreateController[0]

File: test-client.py
#!/usr/bin/env python
import dbus

#def getObject(bus, name, interface)
#  res = dbus.Interface(o, interface)
#  return res

bus = dbus.SessionBus()
xanguliIntro = dbus.Interface(bus, '/Xanguli', 'org.freedesktop.DBus.Introspectable')
print xanguliIntro.introspect
xanguliXanguli = dbus.Interface(bus, '/Xanguli', 'org.xanguli.Xanguli')
print 'fik xanguli'

